redash/query_runner/mongodb.py

```diff
@@ -184,19 +191,20 @@ def _get_collection_fields(self, db, collection_name):
         # as we don't know the correct order. In most single server installations it would be
         # find. In replicaset when reading from non master it might not return the really last
         # document written.
-        first_document = None
-        last_document = None
-
-        for d in db[collection_name].find().sort([("$natural", 1)]).limit(1):
-            first_document = d
-
-        for d in db[collection_name].find().sort([("$natural", -1)]).limit(1):
-            last_document = d
+        collection_is_a_view = self._is_collection_a_view(db, collection_name)
+        documents_sample = []
+        if(collection_is_a_view):
```

Comment:
You don't need parentheses around the `if` statement:

```suggestion
        if collection_is_a_view:
```

Reply:
Cool!